### PR TITLE
Included the latest scalability test and automation scripts

### PR DESCRIPTION
**Pull request by chandrams**

### CODE REVIEW

File: tests/scripts/remote_monitoring_tests/scale_test/rosSimulationScalabilityTest.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+import copy
+import datetime
+import json
+import time
+
+import requests
+
+
+def loadData():
+    createdata = {"version":"1.0","experiment_name":"quarkus-resteasy-kruize-min-http-response-time-db_10","cluster_name":"cluster-one-division-bell","performance_profile":"resource-optimization-openshift","mode":"monitor","target_cluster":"remote","kubernetes_objects":[{"type":"deployment","name":"tfb-qrh-deployment_5","namespace":"default_5","containers":[{"container_image_name":"kruize/tfb-db:1.15","container_name":"tfb-server-0"},{"container_image_name":"kruize/tfb-qrh:1.13.2.F_et17","container_name":"tfb-server-1"}]}],"trial_settings":{"measurement_duration":"15min"},"recommendation_settings":{"threshold":"0.1"}}
```
Comment (msvinaykumar) [inline tests/scripts/remote_monitoring_tests/scale_test/rosSimulationScalabilityTest.py:11]: should we move this to File ?

File: tests/scripts/remote_monitoring_tests/scalability_test.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+For example,
+
+```
+<AUTOTUNE_REPO>/tests/scripts/remote_monitoring_tests/scale_test/remote_monitoring_scale_test_bulk.sh -i quay.io/kruize/autotune_operator:0.0.20_mvp  -u 250  -d 15  -n 20 -t 6  -q 10  -s 2023-01-10T00:00:00.000Z  -r /tmp/scale_test_results
+
+```
+
+Once the tests are complete, manually check the logs for any exceptions or errors or crashes.
```
Comment (msvinaykumar) [inline tests/scripts/remote_monitoring_tests/scalability_test.md:48]: also mention how to get data using sh command on Logs files and DB commands

File: tests/scripts/remote_monitoring_tests/scale_test/rosSimulationScalabilityTest.py (hunk 1 above)
Comment (chandrams) [reply tests/scripts/remote_monitoring_tests/scale_test/rosSimulationScalabilityTest.py]: Done

**Review by msvinaykumar — APPROVED** [review]
LGTM

**Comment by chandrams** [comment]
@dinogun - Can you please review, thank you.

**Pull request merged**